support/src/main/java/bisq/support/moderator/ModeratorService.java

```diff
@@ -156,11 +156,25 @@ public void deleteReportToModeratorMessage(ReportToModeratorMessage reportToMode
         persist();
     }
 
-    public CompletableFuture<BroadcastResult> banReportedUser(ReportToModeratorMessage message) {
+    public CompletableFuture<BroadcastResult> banReportedUser(ReportToModeratorMessage message, String banReason) {
         UserIdentity selectedUserIdentity = userIdentityService.getSelectedUserIdentity();
         KeyPair keyPair = selectedUserIdentity.getNetworkIdWithKeyPair().getKeyPair();
-        BannedUserProfileData data = new BannedUserProfileData(message.getAccusedUserProfile(), staticPublicKeysProvided);
```

Comment:
NIT: Move selectedUserIdentity and keyPair closer to its usage. Makes method easier readable if the 2 tasks are clearly grouped.